templates/edge-auth-dep.yml

```diff
@@ -45,7 +45,7 @@ spec:
 # Update for your configuration:
           - name: client_secret # this can also be provided via a secret named of-client-secret
             value: ""
-          - name: client_id
+         - name: client_id
```

Comment:
This needs to be fixed. The space was ok

Reply:
Please can you take that?